alerter.c

```diff
@@ -26,6 +26,8 @@ void alertInCelcius(float farenheit) {
 int main() {
     alertInCelcius(400.5);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold. i.e. if threshold is 392 F (i.e. 200 c) then consider values <392, =392 and >392